applications/GeoMechanicsApplication/custom_utilities/functions_for_parameters.h

```diff
@@ -0,0 +1,37 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//                   Marjan Fathian,
+//                   Gennady Markelov
+//
+
+#pragma once
+
+#include "includes/kratos_parameters.h"
+
+namespace Kratos {
+
+    class FunctonsForParameters
+    {
+    public:
+
+            static Parameters ExtractParameters(const Parameters & rSourceParameters,
+                const std::vector<std::string>&rNamesOfParametersToCopy);
+
+            static void AppendParameterNameIfExists(const std::string& rParameterName,
+                const Parameters& rSourceParameters,
+                std::vector<std::string>& rResult);
+
+            static bool HasTableAttached(const Parameters& rSettings);
+
+            static bool HasTableAttached(const Parameters& rSettings, int component);
+    };
+
+}
```

Comment:
Also here, no indentation is needed after namespace (only for the class itself)

Reply:
It is done.